miio/huizuo.py

```diff
@@ -25,8 +35,8 @@ class HuizuoException(DeviceException):
     pass
 
 
-class HuizuoStatus:
-    def __init__(self, data):
+class HuizuoMiotStatus:
```

Comment:
```suggestion
class HuizuoStatus:
```
This is not needed IMO. I assume there will be no miio version.

Reply:
Changed everywhere